spec/live/ticket_spec.rb

```diff
@@ -27,6 +27,39 @@ def valid_attributes
   it_should_be_readable agent, :ccd_tickets, create: true
   it_should_be_readable organization, :tickets
 
+  describe "#create" do
+    context "when passing large objects as parameters" do
+      let(:requester) { client.users.search(query: 'role:end-user').reject { |user| user.photo.nil? }.first }
+      let(:organization) { client.organizations.sample }
+      let(:ticket_parameters) do
+        {
+          subject: 'live spec subject',
+          description: 'live spec description',
+          requester: requester,
+          organization: organization
+        }
+      end
+
+      before do
+        VCR.use_cassette("ticket_create_with_objects") do
+          @ticket = ZendeskAPI::Ticket.create(client, ticket_parameters)
+        end
+      end
+
+      it 'should be creatable' do
+        expect(requester).to_not be_nil
+        expect(@ticket).to_not be_nil
+        expect(@ticket.send(:id)).to_not be_nil
+      end
+    end
+
+    after do
+      VCR.use_cassette("ticket_destroy_with_objects") do
+        @ticket.destroy!
+      end
+    end
```

Comment:
If the ticket is not created we would have a confusing double error, which we can avoid with this:

```suggestion
    after do
      return unless @ticket
      VCR.use_cassette("ticket_destroy_with_objects") do
        @ticket.destroy!
      end
    end
```